src/app.js

```diff
@@ -151,15 +153,29 @@ class Messenger extends EventEmitter {
     this.app.get('/ping', (req, res) => {
       res.send('pong');
     });
+
+    this.port = config.get('port');
+    this.isListening = 0;
+    this.start();
   }
 
   start() {
```

Comment:
I like this. Going to implement it and see how it works. The complexity it adds isn't that bad compared to the PR.